mln/services/inventory.py

```diff
@@ -35,6 +36,10 @@ def remove_inv_item(user, item_id, qty=1):
 	else:
 		raise RuntimeError("%s has fewer items than the %i requested to delete" % (stack, qty))
 
+def return_invalid_modules(user): 
+	for module in user.modules.filter(Q(pos_x__isnull=True) | Q(pos_y__isnull=True)): 
+		add_inv_item(user, module.item_id)
+		module.delete()
```

Comment:
It probably won't matter much for this rare case, but you can save some queries by calling `.delete()` on the whole query object instead of each module, like [here](https://github.com/MellonNet/mln-backend-emulator/blob/master/mln/services/page.py#L26).